proxy/interceptor.go

```diff
@@ -2884,6 +2939,64 @@ func visitFailures(ctx *VisitFailuresContext, options *VisitFailuresOptions, obj
 				return err
 			}
 
+		case *command.ScheduleNexusOperationCommandAttributes:
```

Comment:
So a bunch of stuff got added to the failure interceptor, which previously I guess did not start from the top of anything that contains a failure, like the payload one does. Is that intentional? Assuming it is, is starting from the top like this a bad thing?

Reply:
I do not think it's a bad thing, on the contrary it is a good thing. I think the switch statements for payloads should work for every proto message that might contain a payload in any way. Similarly, I think the switch statements for failures should work for every proto message that might contain a failure in any way.

But @Quinn-With-Two-Ns's other comment alter is right, some of these seem to be calling `visitFailures` with no messages to visit.

Reply:
Great, yeah that's what I thought too. I'll sort out the visit with nothing problem